os-cinder.te

```diff
@@ -17,11 +17,11 @@ gen_require(`
 allow httpd_t cinder_log_t:file { open create };
 allow httpd_t cinder_log_t:dir { add_name write };
 
-# Bugzilla 1820504
-allow httpd_t cinder_var_lib_t:dir { add_name write create };
+# Bugzilla 1820504 and 2053849
+allow httpd_t cinder_var_lib_t:dir { add_name write create remove_name };
 
 # Allow httpd to handle files in statedir
-allow httpd_t cinder_var_lib_t:file { open create append getattr lock };
+allow httpd_t cinder_var_lib_t:file { open create append getattr lock ioctl unlink};
```

Comment:
Likewise, we are slowly adding all of the permissions to manage files so I think it would make sense to replace this with the manage_files pattern directly. It seems there were read denials in the logs as well, which would then also be covered:

`manage_files_pattern(httpd_t, cinder_var_lib_t, cinder_var_lib_t);`